assets/csidriveroperators/ovirt/07_deployment.yaml

```diff
@@ -77,9 +77,14 @@ spec:
             requests:
               memory: 50Mi
               cpu: 10m
+          nodeSelector:
+            node-role.kubernetes.io/master: ""
           tolerations:
             - key: CriticalAddonsOnly
               operator: Exists
+            - key: node-role.kubernetes.io/master
+              operator: Exists
+              effect: "NoSchedule"
           args:
```

Comment:
this was a problem in original template too. The toleration and selector belong to pod's template not to container.

Reply:
I believe this has been updated . I went ahead and moved the other toleration, as I saw you had done this in your PR to the ovirt operator itself.